unity-client/Assets/Scripts/MainScripts/DCL/Controllers/PhysicsSyncController/PhysicsSyncController.cs

```diff
@@ -24,5 +24,6 @@ public void Sync()
 
         isDirty = false;
         Physics.SyncTransforms();
+        Physics.Simulate(Time.fixedDeltaTime);
```

Comment:
IMPORTANT!!!

Changes to the physics controller

Reply:
I don't understand why other physics collisions were working if that line was missing XD, since we aren't doing the manual `Physics.Simulate` anywhere else, I'd expect the character to go through every collider, prior to this PR

Reply:
Nice catch. I believe we aren't using the OnTrigger events elsewhere? This may be the reason this wasn't needed.